model/qti/metadata/imsManifest/MetaMetadataExtractor.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2024 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoQtiItem\model\qti\metadata\imsManifest;
+
+use DOMDocument;
+use DOMXPath;
+use InvalidArgumentException;
+
+class MetaMetadataExtractor
+{
+    private const NAMESPACE_LOM = 'http://ltsc.ieee.org/xsd/LOM';
+    private const NAMESPACE_DEFAULT = 'http://www.imsglobal.org/xsd/imscp_v1p1';
+    // phpcs:disable Generic.Files.LineLength
+    private const META_METADATA_PROPERTIES_QUERY_STRING = '//imsmd:metaMetadata/default:extension/default:customProperties/default:property';
```

Comment:
AFAIK breaking it in two lines would also be legit
```suggestion
    private const META_METADATA_PROPERTIES_QUERY_STRING =
    '//imsmd:metaMetadata/default:extension/default:customProperties/default:property';
```